src/components/GoodList/index.js

```diff
@@ -0,0 +1 @@
+export * from './GoodList';
```

Comment:
Consider renaming this file to `index.ts` to reflect the use of TypeScript, as per the task requirements.